src/pages/video-test.tsx

```diff
@@ -174,6 +177,38 @@ const NotesTabContent: React.FC<{cues: VTTCue[]}> = ({cues}) => {
   )
 }
 
+const PlayerContainer: React.ForwardRefExoticComponent<any> = React.forwardRef<
```

Comment:
I made the container a component so we could dynamically add classes, similar to `Player` - actually just cut and pasted most of those classes, we might make these different? You get the idea.